src/platform/silabs/NetworkCommissioningWiFiDriver.cpp

```diff
@@ -23,6 +23,9 @@
 
 #include <limits>
 
+#define VAL(str) #str
+#define TOSTRING(str) VAL(str)
```

Comment:
Why this double macro indirection?
Also why do we need a macro at all? make the `define` be a string: put the quotes in it.